tests/templates/test_subroutines/test_reflection.py

```diff
@@ -164,11 +164,10 @@ def test_lightning_qubit(self):
 
     @pytest.mark.autograd
     @pytest.mark.parametrize("shots", [None, 50000])
-    @pytest.mark.parametrize("device", ["default.qubit", "default.qubit.legacy"])
```

Comment:
Is there a specific reason why this was also tested with `default.qubit.legacy`?

Reply:
probably just sanity checks to make sure nothign wonky is going on.